src/components/ForkInfo.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <b-card v-if="graphData" title="Fork Statistics">
+        <b-card-text v-if="currentLongestFork != null" class="fork-info" style="font-size: 1.25em">
+            Current Longest Fork: {{currentLongestFork}}
+        </b-card-text>
+        <generic-graph :chart-data="chartData" :custom-options="customOptions"></generic-graph>
+    </b-card>
+</template>
+
+<script>
+    import {last} from 'lodash';
+    import GenericGraph from './GenericGraph'
+    import Helper from '../js/Helper'
+    import {BCard, BCardText} from 'bootstrap-vue/es/components'
+
+    export default {
+        name: "ForkInfo",
+        components: {
+            GenericGraph, BCard, BCardText
+        },
+        props: {
+            graphData: Array
+        },
+        computed: {
+            currentLongestFork: {
+                get() {
+                    if (this.graphData && this.graphData.length > 0)
+                        return last(this.graphData)['value']['numForks'];
+                    else
+                        return null
+                }
+            },
+            chartData: {
+                get() {
+                    return {
+                        datasets: [
+                            {
+                                label: "Fork Count",
+                                data: Helper.transformToChartJSData(this.graphData, ["timestamp"], ["value", "numForks"]),
+                                pointRadius: 0
+                            },
+                            {
+                                label: "Longest Fork",
```

Comment:
Longest fork should not share the same y-axis with fork count because their magnitude may be very different. We could have e.g. 1 fork only but forked a long time ago.

However, historical data for fork count and the longest fork seems to have no meaning to user. But please try to separate the y-axes while I confirm this with Yi.

Reply:
A user can toggle which data sets are displayed by clicking on the label. That should fix the scale of the axis. Is that not sufficient?

Reply:
Didn't realize this functionality. Thanks for telling me.

Reply:
Confirmed with Yi that showing the trend for fork count and longest fork can be helpful. No changes required.